db-init/src/main/resources/database/migrations/V4.7__Add_Claim_FK_On_Claim_Submission.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TABLE claim_submission
+    ADD COLUMN IF NOT EXISTS claim_id UUID DEFAULT uuid_generate_v4 () NOT NULL;
```

Comment:
If it's null, it shouldn't generate a new random UUID. It should use the UUID from the claim table.